osgi.enroute.web.simple.provider/.gitignore

```diff
@@ -1,3 +1,4 @@
 /generated/
 /bin/
 /bin_test/
+.settings/
```

Comment:
can we create this folder without symlink because it interferes with commit process after check out and it might not work in windows machines?